plugins/psx/src/psx-plugin.vala

```diff
@@ -0,0 +1,32 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class Games.Psx : Object, Plugin {
+	private const string FINGERPRINT_PREFIX = "psx";
+	private const string MIME_TYPE = "application/x-cue";
+	private const string MODULE_BASENAME = "libretro-psx.so";
+	private const bool SUPPORTS_SNAPSHOTTING = false;
+
+	public GameSource get_game_source () throws Error {
+		var query = new MimeTypeTrackerQuery (MIME_TYPE, game_for_uri);
+		var connection = Tracker.Sparql.Connection.@get ();
+		var source = new TrackerGameSource (connection);
+		source.add_query (query);
+
+		return source;
+	}
+
+	private static Game game_for_uri (string uri) throws Error {
```

Comment:
You probably don't want to create a game for any CUE sheet, we would need to check if they really represent PlayStation games and if they don't: return null or throw an error (I don't remember the current convention, it's probably throwing an error).